test/test_mps.py

```diff
@@ -79,7 +79,7 @@ def _testRelu(self, np_features, device):
         py_relu = torch.nn.ReLU(inplace=False)(py_tensor)
         py_relu_cpu = py_relu.to("cpu")
 
-        torch.testing.assert_allclose(np_relu, py_relu_cpu)
+        self.assertEqual(np_relu, py_relu_cpu)
```

Comment:
Comparing `numpy.ndarray`'s to `torch.Tensor`'s is not supported in `torch.testing.assert_close`. `self.assertEqual` still allows it.